media-library/overview/upload-files.md

```diff
@@ -52,3 +52,41 @@ You can share a folder with a restricted user by providing "Can contribute" or "
 ## Using the Upload API
 
 You can also upload images to ImageKit.io's media library programmatically. Read the Upload API documentation [here](../../api-reference/upload-file-api/).
+
+## Upload settings
+
+* Extensions
+
+Extensions let you perform certain advanced operations on your assets during upload. For reference about extensions [read here](../../extensions/overview)..
+
+* Custom focus area
+
+Define an important area in the image during upload. This is an optional field and is only relevant for image type files.
```

Comment:
add link to the custom focus area page -> https://docs.imagekit.io/media-library/overview/custom-focus-area